README.md

```diff
@@ -111,28 +113,16 @@ resource "aws_security_group" "security_group" {
 ...
 ```
 
-__A resource's NAME should be the same as the TYPE minus the provider.__
+__A resource's NAME should be the same as the TYPE minus the provider, with a minimalistic identifier as prefix.__
```

Comment:
Would be nice to convey the idea that the identifier should not only be short but also clearly states the purpose of this resource :)